code/game/gamemodes/objective_holder.dm

```diff
@@ -59,9 +59,11 @@
  * Replace old_objective with new_objective
  */
 /datum/objective_holder/proc/replace_objective(datum/objective/old_objective, datum/objective/new_objective)
-	new_objective = add_objective(new_objective, add_to_list = FALSE)
+	new_objective = add_objective(new_objective, add_to_list = FALSE, should_find_target = FALSE)
 	new_objective.owner = old_objective.owner
 	new_objective.team = old_objective.team
```

Comment:
More so a question for the existing code. Why are these not parameters to the objective's `New`? It would make sense to create an objective with the owner and team known. I saw that team already is a parameter, yet owner is not...

Reply:
Yeah I can do that